packages/api-server/src/middleware/withFunctions.ts

```diff
@@ -66,42 +72,47 @@ export const setLambdaFunctions = async (foundFunctions: string[]) => {
   })
 }
 
-const lambdaRequestHandler = async (req: Request, res: Response) => {
+interface LambdaHandlerRequest extends RequestGenericInterface {
+  Params: {
+    routeName: string
+  }
+}
+
+// This will take a express request
+// Then convert it to a lambdaEvent, and pass it to the the approrpiate hanlder for the routeName
+// The LAMBDA_FUNCTIONS lookup has been populated already by this point
+const lambdaRequestHandler = async (
+  req: FastifyRequest<LambdaHandlerRequest>,
+  reply: FastifyReply
+) => {
   const { routeName } = req.params
+
   if (!LAMBDA_FUNCTIONS[routeName]) {
     const errorMessage = `Function "${routeName}" was not found.`
     console.error(errorMessage)
-    res.status(404)
+    reply.status(404)
 
     if (process.env.NODE_ENV === 'development') {
       const devError = {
         error: errorMessage,
         availableFunctions: Object.keys(LAMBDA_FUNCTIONS),
       }
-      res.json(devError)
+      reply
+        .header('Content-Type', 'application/json; charset=utf-8')
```

Comment:
is this explicit header needed? I think it is set automatically if trying to send json object

Reply:
Wasn't sure! Will fastify automatically convert to json if I give it an object?

Reply:
Yes, I believe so.